### ISSUE TITLE
On Assessments, each new page of survey should focus on question 1

### ISSUE THREAD

**Issue by infosec-alchemist** [unfetter-discover/unfetter#807]
If we are using the keyboard shortcuts to move through the form, every time a new page comes up (continue, back, etc), the keyboard focus should be on the first question, not the last button.

I don't know how hard this is to do, so consider low priority.
